impeller/renderer/backend/gles/blit_pass_gles.cc

```diff
@@ -130,14 +130,14 @@ bool BlitPassGLES::OnCopyTextureToBufferCommand(
 bool BlitPassGLES::OnCopyBufferToTextureCommand(
     BufferView source,
     std::shared_ptr<Texture> destination,
-    IPoint destination_origin,
+    IRect destination_region,
     std::string label,
     uint32_t slice) {
   auto command = std::make_unique<BlitCopyBufferToTextureCommandGLES>();
   command->label = label;
   command->source = std::move(source);
   command->destination = std::move(destination);
-  command->destination_origin = destination_origin;
+  command->destination_region = destination_region;
   command->label = label;
   command->slice = slice;
```

Comment:
Here and elsewhere, we should check that slices are from 0 to 5. And, if not, either return false or clamp to the values. I prefer the former.

Reply:
I added this check it blit_pass.cc and a test case.